### PR TITLE
Updated playservices and versions

### PR DESCRIPTION
**Pull request by bartbergmans**
Updated services to prevent mixed versions of google play services

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -1,5 +1,8 @@
 apply plugin: 'com.android.library'
 apply plugin: 'jacoco-android'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group='com.github.bartbergmans'
```
Comment (yayaa) [inline library/build.gradle]: Please remove these changes

**Comment by stevyhacker** [comment]
The play services really are so outdated, please update.
Current   googlePlayServices: 'com.google.android.gms:play-services-location:11.8.0'
Lib uses  googlePlayServices: 'com.google.android.gms:play-services-location:9.6.1'

**Comment by yayaa** [comment]
Sorry that I was not really focused on this lately.
I just quickly checked, looks like travis having a problem with Api 27
@bartbergmans could you try out the recommended solution [here](https://github.com/travis-ci/travis-ci/issues/8874#issuecomment-350350607) once it's green, we can merge it 👍

**Comment by yayaa** [comment]
Latest release is 15.0.1 😮

**Comment by chrisfraser** [comment]
The travis build failed due to licences not being accepted.

**Comment by yayaa** [comment]
Considering this is very outdated version - I'm closing this PR.

**Pull request closed without merging**